__tests__/utils/inflectors.ts

```diff
@@ -9,36 +13,37 @@ describe("#titleize", () => {
     const res = titleize("_hello__world!_");
     expect(res).toBe("Hello World!");
   });
-  test("does not remove hyphens", () => {
+  test("removes hyphens", () => {
     const res = titleize("hello-world");
-    expect(res).toBe("Hello-World");
+    expect(res).toBe("Hello World");
```

Comment:
We should probably add a test for the new `breadcrumbTitleize`

Reply:
I took it back out.  The place I thought was leaving hyphens in was actually removing them.  Meaning every place in the code base was removing hyphens after calling this function.